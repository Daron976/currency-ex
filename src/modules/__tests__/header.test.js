import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from '../header';

describe('Header', () => {
  test('Header items renders as expected', async () => {
    const snapHeader = renderer.create(
      <BrowserRouter>
        <Header />
      </BrowserRouter>,
    )
      .toJSON();
    expect(snapHeader).toMatchSnapshot();
  });
});
