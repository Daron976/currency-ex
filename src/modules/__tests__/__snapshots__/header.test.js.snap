// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Header items renders as expected 1`] = `
<header
  className="header"
>
  <i
    className="fa-solid fa-backward-step"
    onClick={[Function]}
  />
  <img
    alt="App logo"
    src="https://img.icons8.com/external-icons-smashing-stocks/68/000000/external-Currencies-stock-market-icons-smashing-stocks.png"
  />
  <a
    href="/continents"
    onClick={[Function]}
  >
    <i
      className="fa-solid fa-house"
    />
  </a>
</header>
`;
